packages/core/src/network/network-peers.ts

```diff
@@ -28,12 +30,14 @@ function nextPing(e: Entry): number {
 }
 
 class NetworkPeers {
-  private peers: Map<string, Entry> = new Map()
-  private ignoredPeers: Entry[] = []
+  private entries: Map<string, Entry> = new Map()
+  private ignoredEntries: Entry[] = []
+  // peers which were denied connection via the HoprNetworkRegistry
+  private deniedEntries: Map<string, Pick<Entry, 'id' | 'origin'>> = new Map()
 
   constructor(
-    existingPeers: Array<PeerId>,
-    private exclude: PeerId[] = [],
+    existingPeers: PeerId[],
+    private excludedPeers: PeerId[] = [],
```

Comment:
We never add anything to `excludedPeers`. Why it's needed then?

Reply:
If we dont use it all, then we can remove it, lets open a different issue for this?

Reply:
I realized it's used in unit tests only. So let's keep that in.